third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.h

```diff
@@ -60,8 +60,6 @@ class TargetInfo : public mlir::triton::TargetInfoBase {
   void assertFail(RewriterBase &rewriter, Location loc, StringRef message,
                   StringRef file, StringRef func, int line) const override;
 
-  bool enableLinearLayout() const override { return false; }
```

Comment:
Can you remove this whole function from TargetInfoBase.h?

Reply:
done